### ISSUE TITLE
Urls in fonts.css file breaks compilation

### ISSUE THREAD

**Issue by sergor5** [nuxt-community/google-fonts-module#8]
When you set fontsPath in `nuxt.config.js` as `fontsPath: '~assets/fonts'` which the documentation says to do it in this way. Then If you look at the console you can see the paths of the files are looking weird

![image](https://user-images.githubusercontent.com/29211821/92282543-7df0d980-ef06-11ea-80bc-13ed55eb83cf.png)

But if you change fontsPath to `fontsPath: 'assets/fonts'`. At least it solves the weirdness of the paths as you can see in the next image's console tab
![image](https://user-images.githubusercontent.com/29211821/92282624-ac6eb480-ef06-11ea-90f1-08f5bd70a1df.png)

The problem solves when I change URLs in the fonts.css file manually as `../fonts/FONT-NAME`. As you can see for the font 'Maven Pro' the issue is solved.
![image](https://user-images.githubusercontent.com/29211821/92282960-6e25c500-ef07-11ea-9a05-35ea7cd7b4a0.png)

So I request to the developer of this module to update the URL generating code.

**Comment by ricardogobbosouza** [comment]
Hi @sergor5 
I cannot reproduce this problem.
Could you provide a repository

**Comment by azrikahar** [comment]
Hi @ricardogobbosouza, I was actually facing this same issue as well. I managed to fix it just by changing the backslashes to forwardslashes, without needing to configure anything else. My wild guess is it's a Windows only issue, unless you also cannot reproduce it on a Windows machine then I might be shooting blanks here. 

Here are the steps and solutions I took in detail, hoping that it helps you track this issue down easier.

## Reproduction Step

Note: The `@nuxtjs/google-fonts` installed in this sample is `1.0.1`, the latest one you had updated hours ago.

1. Create a new project using `yarn create nuxt-app my-app`. The selections are the bare minimum as seen below:

    ```bash
    create-nuxt-app v3.2.0
    ✨  Generating Nuxt.js project in test-app
    ? Project name: test-app
    ? Programming language: JavaScript
    ? Package manager: Yarn
    ? UI framework: None
    ? Nuxt.js modules: (Press <space> to select, <a> to toggle all, <i> to invert selection)
    ? Linting tools: (Press <space> to select, <a> to toggle all, <i> to invert selection)
    ? Testing framework: None
    ? Rendering mode: Universal (SSR / SSG)
    ? Deployment target: Server (Node.js hosting)
    ? Development tools: None
    ```

2. Add google-font-module as instructed:
   - `yarn add --dev @nuxtjs/google-fonts`
   - add `'@nuxtjs/google-fonts'` in `buildModules` of `nuxt.config.js`

3. Add top level option for `googleFonts` into `nuxt.config.js` using the following:

```js
googleFonts: {
    families: {
      Quicksand: true,
      'Maven Pro': true,
    },
    download: true
  },
```

4. Run `yarn dev`.

5. Get the following error in terminal:

    ![WindowsTerminal_Gk1w0EM5xf](https://user-images.githubusercontent.com/42867097/92674962-ffa29600-f350-11ea-8bb2-dd97a5ed1f00.png)

## Diagnosis

I suspect this is only happening for Windows machines due to the backslashes, though I might be wrong. However, if you observe the following screenshot, you can see `\f` and `\M` are syntax highlighted differently, which might indicate they are interpreted as escaped (or vice versa) characters, thus causing this `assetsonts` problem (means the `\f` is stripped/escaped).

![Code_dibHT4YcYF](https://user-images.githubusercontent.com/42867097/92675658-84da7a80-f352-11ea-9e26-342597fb4084.png)

## Solution

If I just change the slashes, it will work properly without any issue. Here is the screenshot, and notice that the `/f` and `/M` are not being highlighted anymore, meaning they are seen as usual string format.

![Code_SW6uGVocmf](https://user-images.githubusercontent.com/42867097/92676511-45149280-f354-11ea-85c4-38e3a774936d.png)

---

@sergor5 Would you mind trying my solution? You'll need to remove the `fontsPath` in the googleFonts option and change `../fonts` to `~assets/fonts` in the `fonts.css` file. And just to confirm, may I know what OS is your machine when facing this issue?

**Comment by ricardogobbosouza** [comment]
Hi @azrikahar
Thanks for the excellent review.
The problem is in windows, I'm already solving ...

**Comment by ricardogobbosouza** [comment]
@azrikahar, @sergor5 
Fixed on v1.0.2
